### DISCUSSION TITLE
M3 Max issue with new Patch

### DISCUSSION THREAD

**Discussion by hasuwini77** [OldUnreal/UnrealTournamentPatches#1563] [General] [answered]
Hi guys,

I was enjoying the new patch for a while without issue, then since I'm using a big screen I wanted to play around with the resolution and try to find the most appropriate one in the list, most likely I chose a "bad resolution" can't remember which one but now everytime I try launching the game I get: 

" Couldn't create SDL window: Window is too large.

History: USDLViewport::ResizeViewport <- UOpenGLRenderDevice::SetRes <- UOpenGLRenderDevice::Init <- USDLViewport::TryRenderDevice <- USDLViewport::OpenWindow <- UGameEngine::Init <- InitEngine <- main" 

I've tried re installing the game rebooting and all but still it won't work... looks like I need to change something but I don't know what.... Hope this helps

Config is Macbook pro 16  M3 Max

**Comment by SeriousBuggie** [comment]
You need edit UnrealTournament.ini.
Look for lines:
WindowedViewportX=
WindowedViewportY=
WindowedColorBits=
FullscreenViewportX=
FullscreenViewportY=
FullscreenColorBits=

It can appear in few places. Find all and set normal values to all of it.

**Reply by hasuwini77** [reply]
There is no UnrealTournament.ini file on MacOs, else if there is, tell me where

**Reply by SeriousBuggie** [reply, marked as answer]
It must be there. but it well hidden. 
Try this one:
`~/Library/Application Support/Unreal Tournament/System`
Or file search by name.

**Reply by hasuwini77** [reply]
File was hidden indeed you were correct from the start! Big thanks

**Comment by hasuwini77** [comment]
I don't think the file is being used imo, but there is a UnrealTournament.ini there, 

The values are set to: 

[WinDrv.WindowsClient]
WindowedViewportX=1024
WindowedViewportY=768
WindowedColorBits=32
FullscreenViewportX=1024
FullscreenViewportY=768
FullscreenColorBits=32

Which "normal value" should I enter there then...


<img width="418" alt="Screenshot 2024-04-16 at 09 28 08" src="https://github.com/OldUnreal/UnrealTournamentPatches/assets/114613121/80699839-6662-4a36-9955-2d0bb40278a3">

**Reply by stijn-volckaert** [reply]
Make sure you edit these properties in the [SDLDrv.SDLClient] section

**Reply by hasuwini77** [reply]
I don't see this section, that's the only other section, I've changed it too but still nothing changed...

Strange

<img width="350" alt="Screenshot 2024-04-16 at 09 37 43" src="https://github.com/OldUnreal/UnrealTournamentPatches/assets/114613121/ee727e4e-b0c7-43b4-9e01-e2295e68cfe0">

**Comment by hasuwini77** [comment]
unfortunately still getting the same error when changing res to 1920 x 1080 for example 


<img width="311" alt="Screenshot 2024-04-16 at 09 31 34" src="https://github.com/OldUnreal/UnrealTournamentPatches/assets/114613121/b1e284c4-2a8b-4b74-b5cf-7a90fb55b2c2">

**Comment by hasuwini77** [comment]
The game was working prior without any "System" folder inside the Application Support so I doubt this is the solution

**Comment by hasuwini77** [comment]
Game was working perfectly and I ruined it by trying to find a better resolution haha ^_^ nerf... damnit! :(

**Comment by hasuwini77** [comment]
I found the file called: "Default.ini" inside "Package content" of the UT app, I changed the value there but yet the same result. Unfortunately the issue remains for me... will try if I can see other files.. it's so frustrating, such a weird bug :/. 

<img width="296" alt="Screenshot 2024-04-16 at 22 38 25" src="https://github.com/OldUnreal/UnrealTournamentPatches/assets/114613121/0f4e125a-d513-4619-96eb-1ccfa6af59b9">

**Comment by hasuwini77** [comment]
Anybody can assist here?

**Comment by SeriousBuggie** [comment]
It must be UnrealTournament.ini, not Default.ini. And it must be with section [SDLDrv.SDLClient].
Try use file search for find it. It can be in hidden folders.

**Comment by hasuwini77** [comment]
You guys were correct from the start all apologies! The files were indeed hidden, so going via the terminal helped here. 
I managed to find the UnrealTournament.ini file and modified the issue here... there is a bug that basically turn the value of "FullscreenViewportX= to a huge number like 100000" if you try and change the resolution in the game... 
It's kind of annoying since when you have a big screen the game looks so tiny I mean you basically need to put your resolution to a big one on your big monitor otherwise you'll end up changing the resolution and the bug would pop up...  
See values here ^^ below 

<img width="308" alt="Screenshot 2024-04-17 at 18 23 26" src="https://github.com/OldUnreal/UnrealTournamentPatches/assets/114613121/178e652b-c575-432d-8fa7-3f67b90a9175">



<img width="808" alt="Screenshot 2024-04-17 at 18 13 33" src="https://github.com/OldUnreal/UnrealTournamentPatches/assets/114613121/d0860ba9-f0cb-4576-bafa-fa821648bed0">

Anyway I've fixed it now thanks to you big thanks! <3 
Thanks for all you guys do!
